File: BMO2/BMO2/Cesar/frmCesar.vb
Public Class frmCesar
    Private Sub BtnCifrar_Click(sender As Object, e As EventArgs) Handles btnCifrar.Click
        Dim al As MetodoCesar = New MetodoCesar()
        txtCyD.Items.Add(al.ABC(txtPalabras.Text))
        txtResultados.Text = al.cambio

    End Sub

    Private Sub BtnDesifrar_Click(sender As Object, e As EventArgs) Handles btnDesifrar.Click
        Dim at As MetodoCesar = New MetodoCesar()
        at.Lim()
        txtCyD.Items.Add(at.DFG(txtPalabras.Text))
        txtResultados.Text = at.cambio
    End Sub

    Private Sub SalirToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles salirToolStripMenuItem.Click
        Application.Exit()
    End Sub

    Private Sub AtbashToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles AtbashToolStripMenuItem.Click
        Dim At As New MetodomnsCesar
        At.Abrir()
        Me.Close()
    End Sub
End Class

File: BMO/BMO/Cesar/frmCesar.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Threading;

namespace BMO.Cesar
{
    public partial class frmCesar : Form 
    {             
        public frmCesar()
        {
            InitializeComponent();           
        }        
        
        private void BtnCifrar_Click(object sender, EventArgs e)
        {
            MetodoCesar al = new MetodoCesar();           
            txtCyD.Items.Add(al.Cifrar(txtPalabras.Text));
            txtResultados.Text = al.cambio;
            lblCyD.Text = "Cifrado";
        }

        private void BtnDesifrar_Click(object sender, EventArgs e)
        {
            MetodoCesar at = new MetodoCesar();
            at.Lim();
            txtCyD.Items.Add(at.Decifrar(txtPalabras.Text));
            txtResultados.Text = at.cambio;
            lblCyD.Text = "Decifrado";
        }

        private void SalirToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }

        private void AtbashToolStripMenuItem_Click(object sender, EventArgs e)
        {
            MetodosmnsCesar ob = new MetodosmnsCesar();
            this.Close();
            ob.AbrirAtbash();
        }
    }
}
